.github/workflows/check.yml

```diff
@@ -0,0 +1,47 @@
+# Based upon:
+#   https://github.com/actions/starter-workflows/blob/main/ci/go.yml
+
+on:
+  pull_request:
+  push:
+    branches: [ $default-branch ]
+
+jobs:
+  check:
+
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.13, 1.14, 1.15]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Download Packages
+        run: go get -t -v ./ably/...
+
+      - name: Vet
+        run: go vet ./ably ./scripts
+
+      - name: Build
+        run: go build -v ./...
```

Comment:
What exactly is being built here? It feels a bit redundant given ably-go is a library not an executable

Reply:
It was in the original Travis file, for starters. 😁 

However, if it's adding no benefit then I am happy to remove it. My assumption was that it a valid thing to run on a check workflow, if nothing else as a smoke test to find out asking Go to `build` blows up in smoke.

Reply:
`go test` effectively includes `go build`, so yes, we can remove it.

`go build` on its own is useful when you want to check the library compiles, but don't want to run the tests. It shouldn't have been in Travis in the first place.